index.html

```diff
@@ -0,0 +1,81 @@
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description"
+        content="It is an Educational, Non-commercial Alejandro xandro2021 and Vyacheslav Hacking-NASSA-with-HTML Microverse Trials HTML/CSS project Landing Page" />
+    <meta name="keywords" content="Microverse Trials, Microverse, xandro2021, Hacking-NASSA-with-HTML" />
+    <meta name="author" content="xandro2021 Hacking-NASSA-with-HTML">
+    <title>Alejandro_Vyacheslav_Microverse_Trials Landing Page</title>
+    <link rel="icon" href="./img/favicon.ico" />
+    <link rel="stylesheet" href="css/style.css">
+    <meta property="og:url"
+        content="https://hacking-nassa-with-html.github.io/Alejandro_Vyacheslav_Microverse_Trials_cool_project/" />
+    <meta property="og:title" content="Alejandro and Vyacheslav Microverse Trials Landing Page" />
+    <meta property="og:description" content="Alejandro and Vyacheslav Microverse Landing Page" />
+    <meta property="og:image"
+        content="https://hacking-nassa-with-html.github.io/Alejandro_Vyacheslav_Microverse_Trial_cool_project/img/ogg-data-Alejandro-Microverse-img.png">
+    <meta property="og:image:width" content="400" />
+    <meta property="og:image:height" content="179" />
+</head>
+
+<body>
+    <header id="header">
+        <a href="#firstScreen"><img id="header-img" alt="logo"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" /></a>
+        <nav id="nav-bar">
+            <ul class="navbBar">
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how-it-works">How it works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="firstScreen">
+        <div id="videoElement">
+            <iframe id="video" class="large-screen" src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+                title="YouTube video player" frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+
+        <div id="formContainer">
+            <form method="POST" id="form" action="https://www.freecodecamp.com/email-submit">
+                <input type="email" id="email" placeholder="#email" name="email" value="" />
+                <input type="submit" id="submit" name="" value="Submit" />
+            </form>
+        </div>
+    </section>
+
+    <section id="features">
+        <h3 class="sectionHeadline">Features</h3>
+    </section>
+    <section id="how-it-works">
+        <h3 class="sectionHeadline">How it Works</h3>
+    </section>
+    <section id="pricing">
+        <h3 class="sectionHeadline">Pricing</h3>
+    </section>
```

Comment:
wrap the rest of the body components into a main tag

Reply:
Same here as mentioned before please give them a reason why they should apply requested changes.

Reply:
Please wrap the rest of the body components into a main tag in order to make it more semantically accurate. 
The rest of the HTML tags are pieces of the main part of the information of the body. In other words, those sections are the main content of this website, so there is an appropriate semantic tag to wrap them, and it is <main></main>